xs/src/libslic3r/PrintObject.cpp

```diff
@@ -1095,4 +1095,55 @@ PrintObject::_infill()
     this->state.set_done(posInfill);
 }
 
+SupportMaterial *
+PrintObject::_support_material()
```

Comment:
Any reason why you're using leading underscores with the names? If you don't want them to conflict with the things exposed in the perl half (and don't want to write XSP entries for them), wrap them in #ifndef SLIC3RXS ... #endif // SLIC3RXS